descartes_planner/include/descartes_planner/sparse_planner.h

```diff
@@ -88,7 +88,7 @@ class SparsePlanner: public descartes_core::PathPlannerBase
   };
 
   double sampling_;
-  int error_code_;
+  mutable int error_code_;
```

Comment:
Generally `mutable` is used to get around code that isn't const correct.  Why is it required here?